torch/nn/modules/module.py

```diff
@@ -461,6 +461,7 @@ def __init__(self) -> None:
         super().__setattr__('_load_state_dict_pre_hooks', OrderedDict())
         super().__setattr__('_load_state_dict_post_hooks', OrderedDict())
         super().__setattr__('_modules', OrderedDict())
+        super().__init__()
```

Comment:
This still doesn't allow you to do things like:
```python
import torch
from torch import nn

class MyClass:
    def __init__(self, foo):
        self.foo = foo


class MyMod(nn.Module, MyClass):
    def __init__(self, foo):
        super().__init__(foo)

m = MyMod(3)
print(m)
```

Could you change the signature to be `    def __init__(self, *args, **kwargs) -> None:` and this call to `super().__init__(*args, **kwargs)` please?

Reply:
done in https://github.com/pytorch/pytorch/pull/91819/commits/e60b22dede072ef04999ca9bd199898976f161ed